x/tally/keeper/abci.go

```diff
@@ -208,7 +208,7 @@ func (k Keeper) FilterAndTally(ctx sdk.Context, req types.Request) (TallyResult,
 		return result, errorsmod.Wrap(err, "error while applying filter")
 	}
 
-	tallyWasm, err := k.wasmStorageKeeper.GetDataRequestWasm(ctx, req.TallyBinaryID)
+	tallyWasm, err := k.wasmStorageKeeper.GetOracleProgram(ctx, req.TallyBinaryID)
```

Comment:
Should this not be `req.TallyProgramID`?

Reply:
There are only oracle programs, it’s just that in the DR you can specify one for the execution phase `exec_program_id` and the tally phase `tally_program_id`